app/src/test/java/homer/model/outlets/OutletTest.java

```diff
@@ -0,0 +1,80 @@
+package homer.model.outlets;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import homer.DeviceInfoImpl;
+import homer.api.DeviceIdImpl;
+import homer.model.lights.Light;
+
+final class OutletTest {
+
+    private final OutletFactory factory = new OutletFactoryImpl();
```

Comment:
is there any reason to not make `outletfactory` a static class? i don't think it needs to maintain state

Reply:
I don't get it, why should the factory be static?

Reply:
> I don't get it, why should the factory be static?

to avoid instantiaing the class, and it makes its purpose more explicit.

Reply:
`static` is an illegal modifier for this class. Even your unit test class `TemeratureTest` isn't `static`, am I doing something wrong?

Reply:
what i meant is that it should be using static methods.

Reply:
The method should be static, not the class

Reply:
I'll fix this and then it should be ready for review i think.

Reply:
Fyi this remained unchanged

Reply:
I still don't understand what's the difference between OutletFactory and TemperatureFactory, what methods should be static?  Can you be more specific please?

Reply:
>what methods should be static? 

the ones that don't need to be keep track of state